CMakeLists.txt

```diff
@@ -3,6 +3,10 @@ file(READ "version" version)
 project(libddwaf VERSION ${version})
 
 set(CMAKE_OSX_DEPLOYMENT_TARGET "10.12" CACHE STRING "Minimum OS X deployment version")
+if(CMAKE_OSX_ARCHITECTURES MATCHES "x86_64" OR
+  CMAKE_OSX_ARCHITECTURES MATCHES "arm64")
+  set(CPU_TYPE ${CMAKE_OSX_ARCHITECTURES})
```

Comment:
This seemed to be required early on, not sure if it still is with subsequent changes.

Reply:
Seems to do fine without it. Also `CPU_TYPE` is not something cmake knows about, so I don't know what effect this could have.